docs/building-open-mpi/required-support-libraries.rst

```diff
@@ -0,0 +1,128 @@
+.. _install-required-support-libraries-label:
+
+Required support libraries
+--------------------------
+
+Open MPI uses the following support libraries:
+
+* `Hardware Locality (hwloc) <https://www.open-mpi.org/projects/hwloc/>`_
+* `Libevent <https://libevent.org/>`_
+* `PMIx <https://pmix.org/>`_
+* `PRRTE <https://github.com/openpmix/prrte>`_
+
+Full copies of the source code of these libraries are bundled in the
+Open MPI source code distribution for end-user convenience.
+
+* In some cases, the presence of these bundled versions can lead to
+  obscure corner cases.  If this has happened to you, continue reading
+  this section.
+* If you have no problems with Open MPI and these four support
```

Comment:
I disagree with this statement.  For end users, it's probably fine, but for anyone installing into /usr or /usr/local, not understanding the impacts of using internal libraries can be problematic for other applications.

Reply:
I'll just remove these 2 bullets.